src/components/UserList/UserList.component.tsx

```diff
@@ -38,11 +60,17 @@ export function UserList(): JSX.Element {
                         {namesMapped}                       
                     </li>
                     <li>
-                        <strong>E-mail:</strong><br/>
+                        <strong>E-MAIL:</strong><br/>
                         {emailMapped}
                     </li>
                 </Ul>
+                <Navegation>
+                    <BTNavegation onClick={previusPage}>Anterior</BTNavegation>
 
+                    <BTNavegation onClick={nextPage} disabled={nextPageexists == false}>
```

Comment:
===

Reply:
quando comparar, 3 iguais força a ser exatamente o que vc quer, entao no geral vc usa 3 e não 2 iguais.
Outra forma que seria mais elegante, já q vc quer que o disabled seja o oposto de existir proxima página seria `disabled={!nextPageExists}`
Outro ponto: o previous não precisaria desse tipo de tratamento tbm? vai querer voltar antes da pagina zero?

Reply:
e o resultado do graphql não é hasPreviousPage/hasNextPage? Pq essa variavel diferente?

Reply:
no caso do previous ele tem esse tratamento no if, pq ele só funciona caso o offset seja >= ao limit

Reply:
Vou adcionar esse tratamento de disabled ao previous para fazer ele sumir na primeira pagina